.gitignore

```diff
@@ -0,0 +1,10 @@
+__pycache__/
+
+# Environments
+.env
```

Comment:
Lot of env added. Hopefully this works? Not a major change though. So we can safely merge it.